tests/h/emails/reply_notification_test.py

```diff
@@ -109,10 +111,10 @@ def test_returns_text_and_body_results_from_renderers(self,
         assert html == 'HTML output'
         assert text == 'Text output'
 
-    def test_returns_subject_with_reply_username(self, notification, pyramid_request):
+    def test_returns_subject_with_reply_display_or_user_name(self, notification, pyramid_request, reply_user):
         _, subject, _, _ = generate(pyramid_request, notification)
 
-        assert subject == 'ron has replied to your annotation'
+        assert subject == '{} has replied to your annotation'.format(reply_user.display_or_user_name)
```

Comment:
This test would still pass if the code used `display_name`, because in our current test fixture the `display_name` and `display_or_user_name` return equivalent values. This feels like there’s something missing, be that an extra test case or a smaller interface.